host-controller/src/main/java/org/jboss/as/host/controller/HostControllerService.java

```diff
@@ -128,6 +130,18 @@ public void start(StartContext context) throws StartException {
             b.append("Configured system properties:");
             for (String property : new TreeSet<String>(properties.stringPropertyNames())) {
                 String propVal = property.toLowerCase(Locale.getDefault()).contains("password") ? "<redacted>" : properties.getProperty(property, "<undefined>");
+                if (property.toLowerCase(Locale.getDefault()).equals("sun.java.command") && !propVal.isEmpty()) {
```

Comment:
Sorry I missed this previously. These should probably be `Locale.ROOT`.

Reply:
Also corrected the other two places.